### DISCUSSION TITLE
Notion API down?

### DISCUSSION THREAD

**Discussion by Galsor** [ramnes/notion-sdk-py#87] [General]
Hello, 

I was running test this morning and I got `httpcore.ConnectError: [Errno -3] Temporary failure in name resolution` while runnig `notion_client`. 
When I tried with Curl I got `curl: (6) Could not resolve host: api.notion.com`. 
Nothing on Twitter about an ongoing shutdown. 
Are you facing similar issues?

**Comment by ramnes** [comment]
Hey @Galsor! I'm not experiencing any issue and it seems that Notion was up when you posted: https://status.notion.so/

**Reply by Galsor** [reply]
OK... Any idea what could be the reason of such issue ?
